sphinx/builders/__init__.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, app):
         self.tags.add(self.name)
         self.tags.add("format_%s" % self.format)
         self.tags.add("builder_%s" % self.name)
+        self.writing = "writing {:}... ".format(self.name)
```

Comment:
Saving a single builder-specific output message to swap into the 3 instances of ``'writing output... '`` below.